validphys2/src/validphys/plotutils.py

```diff
@@ -25,13 +26,59 @@
 
 log = logging.getLogger(__name__)
 
+def subplots(figsize=None,nrows=1, ncols =1,sharex=False, sharey=False):
+    """
+    Use matplotlib.figure.Figure() objects and not import pyplot anywhere. 
+    The reason is that pyplot maintains a global state that makes it misbehave 
+    in multithreaded applications such when executed under dask parallel mode.
```

Comment:
```suggestion
    Use matplotlib.figure.Figure() objects to avoid importing ``pyplot`` anywhere. 
    The reason is that ``pyplot`` maintains a global state that makes it misbehave 
    in multithreaded applications such as when executed under dask parallel mode.
```

Add also an explanation of what the function does (same for the one above). I guess something like "Wrapper to generate subplots using matplotlib.figure...." would be enough. Same for the function below.